src/pages/[platform]/tools/libraries/configure-categories/index.mdx

```diff
@@ -26,10 +19,207 @@ export function getStaticProps(context) {
 
-<Fragments fragments={{ 'react-native': reactnative0 }} />
\ No newline at end of file
+## General configuration
+
+The `Amplify.configure()` configuration can generally be included in your project _entrypoint_ file (i.e. _/src/main.js_). It takes in two parameters - `ResourcesConfig` and `LibraryOptions`.
+
+```ts
+Amplify.configure(
+  {}: ResourcesConfig;
+  {}?: LibraryOptions;
+);
+```
+
+## Authentication (Amazon Cognito)
+
+```js
+Amplify.configure({
+  Auth: {
+    Cognito: {
+      userPoolClientId: string;
+      userPoolId: string;
+      userPoolEndpoint?: string;
+      signUpVerificationMethod?: 'code' | 'link';
+      loginWith?: {
+        oauth?: OAuthConfig;
+        username?: boolean;
+        email?: boolean;
+        phone?: boolean;
+      };
+    }
+  }
+});
+```
+
+## Storage (Amazon S3)
+
+```js
+Amplify.configure({
+  Storage: {
+    S3: {
+      bucket?: string;
+      region?: string;
+    }
+  }
+});
+```
+
+## GraphQL API (AWS AppSync)
+
+```js
+Amplify.configure({
+  API: {
+    GraphQL: {
+      endpoint: string;
+      region?: string;
+      apiKey?: string;
+      customEndpoint?: string;
+      customEndpointRegion?: string;
+      defaultAuthMode: GraphQLAuthMode;
+      modelIntrospection?: ModelIntrospectionSchema;
+    }
+  }
+});
+```
+
+## REST API (Amazon API Gateway)
+
+```js
+Amplify.configure({
+  API: {
+    REST: [
+      'your-api-name':{
+        endpoint: string;
+        region?: string;
+        service?: string; //Optional service name string to sign the request with IAM credentials. Default 'execute-api'
+      }
+    ]
+  }
+});
+```
+
+## Predictions
+
+```js
+Amplify.configure({
+  Predictions: {
+    convert: {
+      translateText: {
+        region: 'us-east-1',
+        proxy: false,
+        defaults: {
+          sourceLanguage: 'en',
+          targetLanguage: 'zh'
+        }
+      },
+      speechGenerator: {
+        region: 'us-east-1',
+        proxy: false,
+        defaults: {
+          VoiceId: 'Ivy',
+          LanguageCode: 'en-US'
+        }
+      },
+      transcription: {
+        region: 'us-east-1',
+        proxy: false,
+        defaults: {
+          language: 'en-US'
+        }
+      }
+    },
+    identify: {
+      identifyText: {
+        proxy: false,
+        region: 'us-east-1',
+        defaults: {
+          format: 'PLAIN'
+        }
+      },
+      identifyEntities: {
+        proxy: false,
+        region: 'us-east-1',
+        celebrityDetectionEnabled: true,
+        defaults: {
+          collectionId: 'identifyEntities8b89c648-test',
+          maxEntities: 50
+        }
+      },
+      identifyLabels: {
+        proxy: false,
+        region: 'us-east-1',
+        defaults: {
+          type: 'LABELS'
+        }
+      }
+    },
+    interpret: {
+      interpretText: {
+        region: 'us-east-1',
+        proxy: false,
+        defaults: {
+          type: 'ALL'
+        }
+      }
+    }
+  }
+});
```

Comment:
I think this is how the other configs should be defined too. Here we are actually creating an example